docs/samples/browser-plugin-meetings/app.js

```diff
@@ -110,7 +102,7 @@ function generateWebexConfig({credentials}) {
         enableMediaNegotiatedEvent: false,
         enableUnifiedMeetings: true,
         enableAdhocMeetings: true,
-        enableTurnDiscovery: turnDiscoveryCheckbox.checked,
```

Comment:
`enableTurnDiscovery` config flag was removed in PR#3261 but then it was incorrectly re-introduced in PR#3262 (probably bad merge conflict), so I'm removing it again, because it's no longer needed, the feature was GA-ed over a year ago